week-1/Homework/extra/2-piping.js

```diff
@@ -16,43 +16,51 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
+function add(a, b) {
+  let sum = a + b;
+  return Number(sum.toFixed(2));
```

Comment:
One interesting question is _where_ to do the rounding here.

In general, we try to round numbers as late as we can, ideally just when we're formatting, rather than intermediate results. If you imagine:
```javascript
multiply(1000, add(1, 0.001))
```
if we round in `add`, the answer is `1000`; if don't, the answer is `1001` - by leaving the rounding to the very end, we get more accurate results.